mailpoet/tests/acceptance/Settings/SubscriberCountShortcodeCest.php

```diff
@@ -11,55 +12,91 @@ class SubscriberCountShortcodeCest {
   const INACTIVE_SUBSCRIBERS_COUNT = 2;
   const UNCONFIRMED_SUBSCRIBERS_COUNT = 3;
   const UNSUBSCRIBED_SUBSCRIBERS_COUNT = 6;
+  const BOUNCED_SUBSCRIBERS_COUNT = 2;
   const SUBSCRIBER_LIST_NAME = 'Single Subscriber List';
-  const SUBSCRIBERS_LIST_NAME = 'Various Subscribers';
+  const SUBSCRIBERS_LIST_NAME_ONE = 'Various Subscribers List One';
+  const SUBSCRIBERS_LIST_NAME_TWO = 'Various Subscribers List Two';
+  const PAGE_TITLE = 'Subscribers Shortcode Page';
+  const PAGE_TEXT = 'Your subscriber count is';
+
+  /** @var SegmentEntity */
+  private $segment1;
+  private $segment2;
 
   public function verifySubscribersShortcodeWithSegments(\AcceptanceTester $i) {
     $i->wantTo('Create page with shortcode of one subscriber and segment');
+
     $segmentFactory = new Segment();
     $segment = $segmentFactory->withName(self::SUBSCRIBER_LIST_NAME)->create();
     $subscriberFactory = new Subscriber();
     $subscriberFactory->withSegments([$segment])->create();
-    $pageTitle = 'Subscribers Shortcode Page';
-    $pageText = 'Your subscriber count is';
-    $pageContent = "$pageText [mailpoet_subscribers_count segments=\"{$segment->getId()}\"]";
-    $i->cli(['post', 'create', '--post_type=page', '--post_status=publish', "--post_title='$pageTitle'", "--post_content='$pageContent'"]);
+    $pageContent = self::PAGE_TEXT . " [mailpoet_subscribers_count segments=\"{$segment->getId()}\"]";
+    $postUrl = $i->createPost(self::PAGE_TITLE, $pageContent);
+
     $i->login();
-    $i->amOnPage('/wp-admin/edit.php?post_type=page');
-    $i->waitForText($pageTitle);
-    $i->clickItemRowActionByItemName($pageTitle, 'View');
-    $i->waitForText($pageTitle);
-    $i->waitForText("$pageText 1");
+
+    $i->amOnUrl($postUrl);
+    $i->waitForText(self::PAGE_TITLE);
+    $i->waitForText(self::PAGE_TEXT . " 1");
   }
 
   public function verifySubscribersShortcodeAllCounts(\AcceptanceTester $i) {
     $i->wantTo('Create page with shortcode of all subscribers but different statuses');
-    $pageTitle = 'Subscribers Shortcode Page';
-    $pageText = 'Your subscriber count is';
-    $pageContent = "$pageText [mailpoet_subscribers_count]";
-    $i->cli(['post', 'create', '--post_type=page', '--post_status=publish', "--post_title='$pageTitle'", "--post_content='$pageContent'"]);
+
+    $segmentFactory = new Segment();
+    $this->segment1 = $segmentFactory->withName(self::SUBSCRIBERS_LIST_NAME_ONE)->create();
+    $this->segment2 = $segmentFactory->withName(self::SUBSCRIBERS_LIST_NAME_TWO)->create();
+    $pageContent = self::PAGE_TEXT . " [mailpoet_subscribers_count] segments=\"{$this->segment1->getId()},{$this->segment2->getId()}\"]";
```

Comment:
You have two closing bracelets here.
[mailpoet_subscribers_count **]** segments=\" ...... "**]**";

Reply:
Hi @triple0t 
But there's also the opening bracelet: `$pageContent = self::PAGE_TEXT . " [mailpoet_subscribers_count]`

Reply:
Ok I removed all 2 of them as they aren't required. Thanks!